### ISSUE TITLE
Options dialog needs a default button option

### ISSUE THREAD

**Issue by craxal** [microsoft/AzureStorageExplorer#8101]
Some Options dialogs may have appropriate default selections (for example, the Conflicts dialog). Keyboard users need to press keys several times just to get through the dialog when these default options are already selected, because none of the dialog buttons are focused. Options dialogs should be able to define a default button to make keyboard navigation faster.

Also, navigation of radio buttons in options should be improved such that tabbing navigates to the radio group only, not each radio button. Arrow keys should be used to change the selected radio button instead.
